lua/wire/client/wire_expression2_editor.lua

```diff
@@ -1837,7 +1837,7 @@ function Editor:Close()
 	timer.Stop("e2autosave")
 	self:AutoSave()
 
-	self:Validate()
+	if self.E2 then self:Validate() end
```

Comment:
This is bad style.
Instead, implement that method as empty in your ZCPU-specific subclass